shared/chat/conversation/input-area/normal/index.desktop.js

```diff
@@ -45,9 +45,13 @@ class ConversationInput extends Component<InputProps> {
     if (!body) {
       return
     }
-    const f = add ? body.addEventListener : body.removeEventListener
-    f('keydown', this._globalKeyDownHandler)
-    f('keypress', this._globalKeyDownHandler)
+    if (add) {
```

Comment:
this was causing story tests to fail -- it was expecting `this` to be bound to `body`. Using `bind` was confusing flow.